src/client/components/ReviewEditorComponent/ReviewEditorComponent.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Button from '../Button/Button';
+import unknownUser from '../../assets/images/portraits/unknown-black.png';
+import './ReviewEditorComponent.style.css';
+
+export default function ReviewEditorComponent({
+  avatarUrl,
+  fullName,
+  placeHolderText,
+  buttonStyle,
+  returnTextState
+}) {
+  const [inputValue, setInputValue] = useState();
+  const onChange = (event) => {
+    setInputValue(event.target.value);
+  };
+  return (
+    <div className="container">
+      <div className="top-part">
+        <div className="profile-picture">
```

Comment:
When you bring in the stars you might need to add a `div `that contains the `profile-picture` and the `user-name`, and keep the stars out. Then with flex you can` justify-content: space-between` to keep the container with the image and the user name on one side and the stars on the other side.

Reply:
Great. will do it. :+1: